datasets/common_voice/README.md

```diff
@@ -43,11 +44,10 @@ languages:
 - mt
 - nl
 - or
-- pa-IN
+- pa
 - pl
 - pt
-- rm-sursilv
-- rm-vallader
```

Comment:
these are valid BCP-47 codes according to wikipedia, we can add them to our list of languages